### ISSUE TITLE
Feature request: Virtual machine drivers only

### ISSUE THREAD

**Issue by silviu-social1st-ro** [ntdevlabs/tiny11builder#288]
Hi there, could you make, please, a virtual machine optimised version?
Lots of components, like drivers, are useless in a virtual machine, you could add only drivers for the major VM Hosts: VirtualBox, virtio, hyper-V, VMWare etc. There are lots of use cases when a VM instance is needed and this requires wasting 20+GB of sttorage for a single process running in that VM.

Thank you!

**Comment by blkhawk** [comment]
Yes I tried doing a tiny11 ISO for vmware but it seems to miss some driver that is not apparent and the vmware tools ISO isn't coming with it either
